### ISSUE TITLE
Firefox: per user/tenant forced dark theme is not applied all through

### ISSUE THREAD

**Issue by sgofferj** [goauthentik/authentik#7381]
**Describe the bug**
I have configured a custom background image for my user as well as my tenant. As the background image is mostly dark, I also set the theme to dark. In Chrome that results in white/light/readable text everywhere, however, in Firefox some text, namely the top right menu and letters in the dashboard tiles, remain in the color of the theme that would be active, i.e. light or dark, according to browser settings. Also, the "Logins and authorizations over the last week (per 8 hours)" graph in the admin dashboard is unreadable. Additionally, when switching from user dashboard to admin dashboard, the background flashes white for like half a second or so which is extremely irritating.

Attribute config (identical for user and tenant):
```yaml
settings:
  theme:
    base: dark
    background: >
      background: url('/media/background.jpg'); filter: blur(2px);
      background-position: center; background-repeat: no-repeat;
      background-size: cover;
```
*Note: /media/background.jpg is the NASA picture of the day pulled by a bash script every day at 00:05 UTC.*

**To Reproduce**

1. Configure custom background and base theme
2. Check in Firefox before sunset

**Expected behavior**
As in Chrome, I expect all text in Firefox to match the dark theme override

**Screenshots**
Firefox
![Screenshot_20231030_090855](https://github.com/goauthentik/authentik/assets/3258466/f82d4716-86a3-4366-9e0a-802bff1a942f)

![Screenshot_20231030_092146](https://github.com/goauthentik/authentik/assets/3258466/845dd816-3892-45fa-b0df-130779d2a6db)

Chrome
![Screenshot_20231030_090955](https://github.com/goauthentik/authentik/assets/3258466/903b8deb-9e1c-4c92-827f-df1d9555595c)

**Logs**
No relevant entries

**Version and Deployment (please complete the following information):**

-   authentik version: 2023.10.2
-   Deployment: docker-compose
 
**Additional context**
None

**Comment by sgofferj** [comment]
Issue persists

**Comment by sgofferj** [comment]
Issue persists

**Comment by sgofferj** [comment]
Seems fixed in 2024.04
